integration-tests/ui/src/test/resources/migration/MIGR_TOOLS/xsk-test-app/xsk-test-app.hdi

```diff
@@ -3,14 +3,17 @@
     "users": [
         "DBADMIN"
     ],
-    "group": "xsk-test-app",
-    "container": "xsk-test-app",
+    "group": "MIGR_TOOLS_XSK-TEST-APP",
+    "container": "MIGR_TOOLS_XSK-TEST-APP",
```

Comment:
I think the `-` symbol is not allowed for the HDI container group and name. Please replace it with `_`.